templates/cli/lib/commands/push.js.twig

```diff
@@ -450,12 +453,54 @@ const pushFunction = async ({ functionId, all, yes } = {}) => {
                     updaterRow.fail({ errorMessage: e.message ?? 'General error occurs please try again' })
             }
         }
+
+        if (deploymentCreated) {
+            try {
+                const deploymentId = response['$id'];
+                updaterRow.update({ status: 'Deploying', end: 'Checking deployment status...' })
+                let pollChecks = 0;
+
+                while (true) {
+                    if (pollChecks >= POLL_MAX_DEBOUNCE) {
+                        updaterRow.update({ end: 'Deployment is taking too long. Please check the console for more details.' })
+                        break;
+                    }
+
+                    response = await functionsGetDeployment({
+                        functionId: func['$id'],
+                        deploymentId: deploymentId,
+                        parseOutput: false
+                    });
+
+
+                    const status = response['status'];
+                    if (status === 'ready') {
+                        updaterRow.update({ status: 'Deployed' });
+                        successfullyDeployed++;
+
+                        break;
+                    } else if (status === 'failed') {
+                        updaterRow.fail({ errorMessage: 'Failed to deploy.' });
```

Comment:
Let's show logs, if possible. If it looks too weird, we can link to Console

Reply:
![image](https://github.com/appwrite/sdk-generator/assets/316103/c5793462-d28b-4c32-9e23-92438f3bbbda)